sympy/geometry/tests/test_ellipse.py

```diff
@@ -332,6 +332,11 @@ def test_construction():
     e4 = Ellipse(Point(0, 0), hradius=1, eccentricity=0)
     assert e4.vradius == 1
 
+    #tests for eccentricity > 1
+    raises(GeometryError, lambda: Ellipse(Point(3, 1), hradius=3, eccentricity=sqrt(3)-sqrt(2) +0.6899))
```

Comment:
```suggestion
    raises(GeometryError, lambda: Ellipse(Point(3, 1), hradius=3, eccentricity = sqrt(3) - sqrt(2) + 0.6899))
```
From where did this 0.6899 came into picture? Can we add some simple tests like with eccentricity 2 or S(3)/2?